### DISCUSSION TITLE
Why Does Neo Tree Show Different Directories for CWD and Root Dir in a Monorepo?

### DISCUSSION THREAD

**Discussion by Demianeen** [nvim-neo-tree/neo-tree.nvim#1254] [Q&A] [answered]
Hello everyone,

I'm using LazyVim with Neo Tree in a monorepo setup, and I'm encountering an issue with how the file explorer displays directories. Our monorepo has only one Git repository at the top. When I open Neo Tree using the 'root dir' option, it only shows one package in the monorepo, even though I opened Vim at the repo's root. In contrast, when I use the 'cwd' (current working directory) option, Neo Tree opens in the root of the entire repo as expected.

I'm trying to understand the difference between 'cwd' and 'root dir' in this context and why there's a discrepancy in how Neo Tree displays the directories. Is there a specific configuration or setting that determines how Neo Tree identifies the root directory in a monorepo setup, especially when there's only one Git repository at the top level?

Any insights or suggestions to resolve this issue would be greatly appreciated!

**Comment by cseickel** [comment]
[Nvim Tree](https://github.com/nvim-tree/nvim-tree.lua) is a different project. Did you mean to start this discussion there or were you actually talking about Neo-tree?

**Comment by Demianeen** [comment]
Sorry just figured out that there is a difference. 
And even more😅 , Lazyvim uses neo-tree by default, so my question is the same. Sorry for inconvenience

@cseickel, do you know what can cause this behaviour?

**Comment by cseickel** [comment, marked as answer]
I think those commands/options are something specific to LazyVim. 

Neo-tree will just use the current working directory by default, unless the command is run with the `dir=` option, in which case it will use that dir as the root. You can find information on how to use the `Neotree` command on the README page: https://github.com/nvim-neo-tree/neo-tree.nvim?tab=readme-ov-file#the-neotree-command
